website/src/data/users.js

```diff
@@ -155,6 +155,15 @@ const users = [
     fbOpenSource: false,
     pinned: false,
   },
+  {
+    title: 'Tasit',
+    description: 'A JavaScript SDK for building native mobile Ethereum dapps with React Native',
```

Comment:
Prettier failed because this line is too long and should be on the next.

Reply:
Ah ok, makes sense. Thanks!